hw/top_darjeeling/templates/toplevel.sv.tpl

```diff
@@ -388,7 +397,10 @@ for rst in output_rsts:
     cg_en = 'cg_' + lpg['clock_connection'].split('clk_')[-1]
   else:
     cg_en = top['clocks'].hier_paths['lpg'] + lpg['clock_connection'].split('.clk_')[-1]
-  rst_en = lib.get_reset_lpg_path(top, lpg['reset_connection'])
+  if lpg['unmanaged_reset']:
+    rst_en =  top['unmanaged_resets']._asdict()[lpg['reset_connection']['name']].rst_en_signal_name
+  else:
+    rst_en = lib.get_reset_lpg_path(top, lpg['reset_connection'])
```

Comment:
I know that `lpg` is just a dict, but maybe it makes sense to check more sensibly to make sure `reset_connection` is defined here? (And I realise that this is unchanged in the commit, but I'm asking because you're touching the line :-D)

Reply:
Do you really think that is needed there? There is a single place, where something is something added to the `lpg` dict:

```top['alert_lpgs'].append({
                'name': lpg_name,
                'clock_group': None if unmanaged_clock else clock_group,
                'clock_connection': clock,
                'unmanaged_clock': unmanaged_clock,
                'unmanaged_reset': is_unmanaged_reset(top, reset_name),
                'reset_connection': primary_reset
            })
```

That's already the full lpg